### PR TITLE
media-tv/gentoo-vdr-scripts: finalize migrate to GLEP 81

### PR DESCRIPTION
**Pull request by madmartin**
version bump to 3.0_rc2

Closes: https://bugs.gentoo.org/781344
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Martin Dummer <martin.dummer@gmx.net>

### CODE REVIEW

File: media-tv/gentoo-vdr-scripts/gentoo-vdr-scripts-3.0_rc2-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	sys-process/wait_on_pid"
+BDEPEND="${DEPEND}"
+
+src_install() {
+	default
+
+	VDR_HOME="$(egethome vdr)"
+	echo "VDR_HOME=${VDR_HOME}"
+	diropts -ovdr -gvdr
+	keepdir "${VDR_HOME}"
```
Comment (ConiKost) [inline media-tv/gentoo-vdr-scripts/gentoo-vdr-scripts-3.0_rc2-r1.ebuild:31]: Is this actually needed? HOME is created by acct-*?

**Review by ConiKost — CHANGES_REQUESTED** [review]

File: media-tv/gentoo-vdr-scripts/gentoo-vdr-scripts-3.0_rc2-r1.ebuild (hunk 1 above)
Comment (madmartin) [reply media-tv/gentoo-vdr-scripts/gentoo-vdr-scripts-3.0_rc2-r1.ebuild:31]: Yes, you are right... this is double stitched.
acct-user/vdr does a `keepdir` first, so we really dont need it here. I will fix that.

File: media-tv/gentoo-vdr-scripts/gentoo-vdr-scripts-3.0_rc2-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	acct-user/vdr"
+RDEPEND="${DEPEND}
+	app-admin/sudo
+	sys-process/wait_on_pid"
+BDEPEND="${DEPEND}"
+
+src_install() {
+	default
+
+	VDR_HOME="$(egethome vdr)"
```
Comment (ConiKost) [inline media-tv/gentoo-vdr-scripts/gentoo-vdr-scripts-3.0_rc2-r1.ebuild:28]: ```local``` please.

**Review by ConiKost — APPROVED** [review]
lgtm

**Pull request closed without merging**